Makefile

```diff
@@ -21,11 +23,14 @@ sedseparator = ''
 endif
 
 build_dir = 'tmp/_output'
+configcli_dest := $(build_dir)/configcli.tgz
 
-build: pkg/apis/kubedirector.bluedata.io/v1alpha1/zz_generated.deepcopy.go | $(build_dir)
-	@echo
-	@echo \* Creating node prep package...
-	tar cfzP tmp/_output/nodeprep.tgz nodeprep
+configcli:  | $(build_dir)
+	@if [ -e $(configcli_dest) ]; then exit 0; fi;                             \
+echo "\* Downloading configcli package ...";                                   \
```

Comment:
We can go ahead and indent these script lines out (with spaces) to pretty things up somewhat, as with other similar stuff below.